Code Styler Issues (dollar operator conversion) with season error message

**_gh4actions_** call-docs-and-styler error styling `SSmase.R and `SSplotHCxval.R` converting dollar operators to brackets.

```
Status	Count	Legend 
1: When processing R/SSmase.R: <text>:149:89: unexpected symbol
148:       Season <- as.numeric(Season)[1]
149:       if (is.na(Season)) stop("Season must a default or and or the integer of indices[["Seas
                                                                                             ^ 
2: When processing R/SSplotHCxval.R: <text>:[41](https://github.com/PIFSCstockassessments/ss3diags/runs/6186033932?check_suite_focus=true#step:13:41)4:89: unexpected symbol
413:       Season <- as.numeric(Season)[1]
414:       if (is.na(Season)) stop("Season must a default or and or the integer of indices[["Seas
                                                                                             ^ 
Error: Error: styler threw an error
Execution halted
✔ 	22	File unchanged.
ℹ 	0	File changed.
✖ 	2	Styling threw an error.
────────────────────────────────────────
Error: Process completed with exit code 1.
```

`SSmase.R` and `SSPlotHCxval.R` throws this error message if a non-default `Season` value is `NA` (or a non-integer):
**stop("Season must a default or and or the integer of indices$Seas 1,2,3,4")**

https://github.com/PIFSCstockassessments/ss3diags/blob/8c76cf069e7985d7b9d60c92d18961757bfb90a2/R/SSmase.R#L147-L150

https://github.com/PIFSCstockassessments/ss3diags/blob/8c76cf069e7985d7b9d60c92d18961757bfb90a2/R/SSplotHCxval.R#L412-L415

 Converting **indices$Seas** to _indices[[Seas]]_ manually is no problem. 

My other question is there a less ambiguous way to say: _Season must a default or and or the integer of indices$Seas 1,2,3,4_.

## Resolution

1. Manually renaming  `indices$Seas ->  indices[[Seas]]` 
2. Replace the error message to **"Season must be "default" or an integer value of a season included in the index data (ie 1,2,3,4)**" .

--- a/R/SSmase.R
+++ b/R/SSmase.R
@@ -146,7 +146,7 @@ SSmase <- function(retroSummary, quants = c("cpue", "len", "age", "con"), Season
       }
     } else {
       if(as.integer(Season) < 1 | as.integer(Season) > 4) stop("Season must be an integer between 1 and 4")
-      if (is.na(Season)) stop("Season must be "default" or an integer value of a season included in the index data (ie 1,2,3,4).")
+      if (is.na(Season)) stop("Season must be 'default' or an integer value of a season included in the index data (ie 1,2,3,4).")
       Season <- as.numeric(Season)[1]
     }
 

--- a/R/SSmase.R
+++ b/R/SSmase.R
@@ -10,7 +10,7 @@
 #' @param models Optional subset of the models described in
 #' r4ss function summaryoutput().  Either "all" or a vector of numbers indicating
 #' columns in summary tables.
-#' @param Season option to specify Season - Default uses first available, i.e. usual Seas = 1
+#' @param Season option to specify Season as an integer of value 1-4 - Default uses first available, i.e. usual Seas = 1
 #' @param endyrvec Optional single year or vector of years representing the
 #' final year of values to show for each model. By default it is set to the
 #' ending year specified in each model.
@@ -145,8 +145,9 @@ SSmase <- function(retroSummary, quants = c("cpue", "len", "age", "con"), Season
         cat("Taking Season", Season, "by default for Index", unique(indices2$Fleet_name))
       }
     } else {
+      if(as.integer(Season) < 1 | as.integer(Season) > 4) stop("Season must be an integer between 1 and 4")
+      if (is.na(Season)) stop("Season must be "default" or an integer value of a season included in the index data (ie 1,2,3,4).")
       Season <- as.numeric(Season)[1]
-      if (is.na(Season)) stop("Season must a default or and or the integer of indices$Seas 1,2,3,4")
     }
 
     indices <- indices[indices$Seas == Season, ]

--- a/R/SSplotHCxval.R
+++ b/R/SSplotHCxval.R
@@ -411,7 +411,7 @@ SSplotHCxval <- function(retroSummary,
       }
     } else {
       if(as.integer(Season) < 1 | as.integer(Season) > 4) stop("Season must be an integer between 1 and 4")
-      if (is.na(Season)) stop("Season must be "default" or an integer value of a season included in the index data (ie 1,2,3,4).")
+      if (is.na(Season)) stop("Season must be 'default' or an integer value of a season included in the index data (ie 1,2,3,4).")
       Season <- as.numeric(Season)[1]
     }
 

--- a/R/SSplotHCxval.R
+++ b/R/SSplotHCxval.R
@@ -10,7 +10,7 @@
 #' @param models Optional subset of the models described in
 #' r4ss function summaryoutput().  Either "all" or a vector of numbers indicating
 #' columns in summary tables.
-#' @param Season option to specify Season - Default uses first available, i.e. usual Seas = 1
+#' @param Season option to specify Season as an integer of value 1-4 - Default uses first available, i.e. usual Seas = 1
 #' @param endyrvec Optional single year or vector of years representing the
 #' final year of values to show for each model. By default it is set to the
 #' ending year specified in each model.
@@ -410,8 +410,9 @@ SSplotHCxval <- function(retroSummary,
         cat("Taking Season", Season, "by default for Index", unique(indices2$Fleet_name))
       }
     } else {
+      if(as.integer(Season) < 1 | as.integer(Season) > 4) stop("Season must be an integer between 1 and 4")
+      if (is.na(Season)) stop("Season must be "default" or an integer value of a season included in the index data (ie 1,2,3,4).")
       Season <- as.numeric(Season)[1]
-      if (is.na(Season)) stop("Season must a default or and or the integer of indices$Seas 1,2,3,4")
     }
 
     indices <- indices[indices$Seas == Season, ]

